deploy/helm/sumologic/values.yaml

```diff
@@ -4022,9 +4026,9 @@ metadata:
           logs:
             receivers:
               - fluentforward
+              - otlp
             processors:
               - memory_limiter
-              - filter/include_fluent_tag_containers
```

Comment:
Will this still work for logs sent by `Fluent Bit`?

Reply:
Yes. This filter only lets through logs from fluent-bit, actually. I'm not entirely sure why it's here, given that we should probably process any record that we get here. @kkujawa-sumo do you know what the purpose was originally?

Reply:
Update: The filter is here to control which logs pass through which pipeline (since they share a receiver), and the fact that they filter out non-fluent-bit logs is a side effect. I'll roll this back and add a separate pipeline for otlp container logs.